cli/src/diff_util.rs

```diff
@@ -2033,12 +2040,13 @@ pub fn show_diff_stats(
         .max()
         .unwrap_or(0);
 
-    let number_padding = max_diffs.to_string().len();
-    // 4 characters padding for the graph
-    let available_width = display_width.saturating_sub(4 + " | ".len() + number_padding);
```

Comment:
I think the intent of `4` is to allocate at least 4 characters for the graph drawings.

BTW, the original PR had 1-space indent as `git` would insert, so I wouldn't be surprised if there was off-by-one error.